### DISCUSSION TITLE
Icon.png is not been adding in Apple Wallet Pass

### DISCUSSION THREAD

**Discussion by akash-86** [alexandercerutti/passkit-visual-designer#24] [Q&A]
Hey, I have gone through your pass visualization website to check the available pass designs. I noticed a small icon on the bottom left side of the pass design. However, when I try to use any icon by setting the icon URL during pass generation, the icon does not appear or append to the pass. This is how I am setting the URL for the icon to "icon.png" -

```javascript
newPass.addBuffer("icon.png", bufferIcon).
```

Is there anything specific that I might be missing here, or is there any concern related to the icon resolution?
<img width="261" alt="Screenshot 2024-01-30 at 1 24 07 PM" src="https://github.com/alexandercerutti/passkit-generator/assets/143791354/49ee58a9-bc18-4501-904e-040e30f9b3bf">

**Comment by alexandercerutti** [comment]
Hi @akash-86, thanks for using both passkit-generator and pkvd!

That's kind of weird. I both tried to add an icon to an empty project and import one that already have it. In both cases I see it. I don't see other medias, but this is another issue.

<img width="274" alt="immagine" src="https://github.com/alexandercerutti/passkit-visual-designer/assets/5442545/0b9a4cc6-4955-41f6-af01-c86e71eea764">

What you should try is to debug your pass (open it as `.zip`) and check if you have an icon inside it. If you don't see it, you should check if there's something wrong with it. If so, open another issue in passkit-generator with your code and all the information needed and I'll try to help you.

**Comment by akash-86** [comment]
Hello @alexandercerutti , in my case the icon.png is being used as icon coming in notification. Whenever I am updating any field in apple pass I am getting the notification with text and the icon which I have shared during the creation of pass in the form of icon.png

**Reply by alexandercerutti** [reply]
@akash-86 But still on your pass you cannot see it in the bottom-left corner, right? So what you were reporting wasn't an issue with PKVD?
